miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/userinfo/UserInfoBridge.kt

```diff
@@ -100,15 +102,29 @@ internal class UserInfoBridge {
                 accessToken.scopes = tokenPermission
                 bridgeExecutor.postValue(callbackObj.id, Gson().toJson(accessToken))
             }
-            val errorCallback = { message: String ->
-                bridgeExecutor.postError(callbackObj.id, "$ERR_GET_ACCESS_TOKEN $message")
+
+            // send a specifically formatted error key to show specific error in mini app
+            val errorCallback = { errorType: MiniAppAccessTokenError ->
+                bridgeExecutor.postError(callbackObj.id, returnAccessTokenError(errorType))
             }
 
             userInfoBridgeDispatcher.getAccessToken(miniAppId, tokenPermission, successCallback, errorCallback)
         } else
             bridgeExecutor.postError(callbackObj.id, "$ERR_GET_ACCESS_TOKEN $ERR_ACCESS_TOKEN_NOT_MATCH_MANIFEST")
     }
 
+    private fun returnAccessTokenError(errorType: MiniAppAccessTokenError): String {
+        return when (errorType.error) {
```

Comment:
I thought, just returning `AccessTokenErrorType.getValue(errorType.error)` would be enough 🤔 
Or, I am not sure about the context of `AuthorizationFailureError` here.

Reply:
for the AuthorizationFailureError and Error type the message will be set by the host app so it needs the case @khairul-alam-licon